public class Cart {
    private double[] prices;
    private final String cartId;
    private int itemCount;

    public Cart(String cartId, int maximumItems) {
        this.cartId = cartId;
        prices = new double[maximumItems];
        itemCount = 0;
    }

    public void addItem(double price) {
        if (itemCount < prices.length) {
            prices[itemCount] = price;
            itemCount++;
        }
    }

    public double getTotal() {
        double total = 0;

        for (int i = 0; i < itemCount; i++) {
            total += prices[i];
        }

        return total;
    }

    public int getItemCount() {
        return itemCount;
    }

    public String getCartId() {
        return cartId;
    }
}